### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by almuitur**

### CODE REVIEW

File: staff/almu-itur/pre-course/6-pasapalabra-html/index.html [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+			<p id="displayResults"></p>
+			<button id="restart" onclick="reset()">VOLVER A JUGAR</button><button id="end" onclick="exitProgram()">EXIT</button>
+
+		</div>
+
+		<div class="ranking">
+			
+			<table id="tableResults">
+				<thead>
+					<trow>
```
Comment (JudithGo) [inline staff/almu-itur/pre-course/6-pasapalabra-html/index.html]: Esta etiqueta trow que es? No la había visto nunca.

Comment (almuitur) [reply staff/almu-itur/pre-course/6-pasapalabra-html/index.html]: Pues creía que era para crear "rows" en una tabla pero parece que está mal, es solo <tr> :(

Comment (JudithGo) [reply staff/almu-itur/pre-course/6-pasapalabra-html/index.html]: ey, puede que no este mal eh... solo preguntaba. Que yo soy una novata como vos.

Comment (aleixcam) [reply staff/almu-itur/pre-course/6-pasapalabra-html/index.html]: ```<trow>``` effectively does not exist, maybe what you're searching is ```<tr>```

File: staff/almu-itur/pre-course/5-calculadora-html/app/app.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+function calculator () {
+
+	var n1 = parseFloat(document.getElementById('n1').value);
+  	var n2 = parseFloat(document.getElementById('n2').value);
+  	var error = false;
+
+  	var operations = [n1 + n2, n1 - n2, n1 * n2, n1 / n2];
+	var resultsRounded = operations.map(function(numberVar) {
+       	if (!Number.isInteger(numberVar)) {
+        	numberVar = numberVar.toFixed(2);
```
Comment (manuelbarzi) [inline staff/almu-itur/pre-course/5-calculadora-html/app/app.js]: avoid re-assigning a value to a function argument (parameter), create a new variable  instead for that

Comment (JudithGo) [reply staff/almu-itur/pre-course/5-calculadora-html/app/app.js]: I thought it would be better to reuse

File: staff/almu-itur/cv/index.html [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
+    <script src="main.js"></script>
+</head>
+<body>
+    <header>
+            <h1>ALMUDENA ITURGOYEN</h1>
+            <h2>Aspiring developer</h2>
+            <h6>Email</h6><h6>Phone</h6>
+    </header>
+        <div class="summary">
```
Comment (Pacoscript) [inline staff/almu-itur/cv/index.html]: Creo que no deberías haber utilizado div, mejor section no?

File: staff/almu-itur/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+               <li>HTML</li>
+               <li>CSS</li>
+               <li>C</li>
+
+           </ul>
+        </div>
+        
+        <div class="languages">
+            <ul>
+                <li><span>Spanish</span>: Native</li>
```
Comment (seand52) [inline staff/almu-itur/cv/index.html]: Why do you use span tags? For future styling/javascript functionality?

File: staff/almu-itur/cv/index.html [hunk 5]
```diff
@@ -0,0 +1,78 @@
+index <!DOCTYPE html>
+<html>
+<head>
+    <meta charset="utf-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <title>CV ALMUDENA ITURGOYEN</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" type="text/css" media="screen" href="main.css" />
```
Comment (maryammdot) [inline staff/almu-itur/cv/index.html]: In this case there is not css file, right?

File: staff/almu-itur/pre-course/6-pasapalabra-html/index.html (hunk 1 above)
Comment (almuitur) [reply staff/almu-itur/pre-course/6-pasapalabra-html/index.html]: Yes, that's right. I'll change it

File: staff/almu-itur/cv/index.html [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+    <script src="main.js"></script>
+</head>
+<body>
+    <header>
+            <h1>ALMUDENA ITURGOYEN</h1>
+            <h2>Aspiring developer</h2>
+            <h6>Email</h6><h6>Phone</h6>
```
Comment (maryammdot) [inline staff/almu-itur/cv/index.html]: Maybe it can be useful to use the tag `<adress>` for this info

File: staff/almu-itur/cv/index.html (hunk 3 above)
Comment (almuitur) [reply staff/almu-itur/cv/index.html]: Sí, voy a cambiarlo. Gracias!

File: staff/almu-itur/cv/index.html (hunk 4 above)
Comment (almuitur) [reply staff/almu-itur/cv/index.html]: I used it to highlight content but I will use strong, instead

File: staff/almu-itur/cv/index.html (hunk 5 above)
Comment (almuitur) [reply staff/almu-itur/cv/index.html]: Correct! I'll delete the line. Thank you!

File: staff/almu-itur/pre-course/5-calculadora-html/app/app.js (hunk 2 above)
Comment (almuitur) [reply staff/almu-itur/pre-course/5-calculadora-html/app/app.js]: Ok, will do

File: staff/almu-itur/cv/index.html (hunk 6 above)
Comment (almuitur) [reply staff/almu-itur/cv/index.html]: Thank you, will do!

Comment (manuelbarzi) [inline staff/almu-itur/cv/index.html]: it is very strange to tag email and phone as headings (h6), may you want to list them or just include them in an address tag?

File: staff/almu-itur/cv/index.html (hunk 3 above)
Comment (manuelbarzi) [reply staff/almu-itur/cv/index.html]: use sections instead

File: staff/almu-itur/cv/index.html (hunk 6 above)
Comment (almuitur) [reply staff/almu-itur/cv/index.html]: Just some information, I am changing it for a p tag

**Pull request merged**